### DISCUSSION TITLE
Guild Icon defaulting for Embed Messages

### DISCUSSION THREAD

**Discussion by squeaky-joe** [discordjs/discord.js#8470] [Q&A]
I am always running into the issue where if the guild doesn't have an iconURL the bot fails and crashes. Can you maybe add something to where if there is no iconURL that is found it defaults to not using one?

**Comment by uShifty** [comment]
I tend to add a default placeholder image/logo for the bot so it could be something like 
`interaction.guild.iconURL() ?? "https://i.imgur.com/image_name.jpeg"`
to prevent the bot from crashing I suggest you do this until they add a placeholder icon which I do not think will happen as that is something we can control.
